commands/create.js

```diff
@@ -18,7 +18,7 @@ const minimist = require('minimist');
 const validatePackageName = require('validate-npm-package-name');
 
 const ENACT_DEV_NPM = '@enact/cli';
-const INCLUDED = path.dirname(require.resolve('@enact/template-sandstone'));
+const INCLUDED = path.dirname(require.resolve('@enact/template-moonstone'));
```

Comment:
Should we abstract this so we can just change the default template in two places instead of a half dozen?

Reply:
I think for now it's fine. Potentially a future task to abstract out all theme-related things in some dynamic way, but for now I think it's fine.